python/tests/conftest.py

```diff
@@ -16,8 +16,8 @@
 
 # Third Party
 import pytest
-from torch.utils.data import DataLoader  # Prevent DataLoader hangs
 from pyspark.sql import SparkSession
+from torch.utils.data import DataLoader  # Prevent DataLoader hangs
```

Comment:
Last time I did this, the DataLoader hangs.  Not sure whether it is still the case this time.